app-frontend/config/webpack/environments/production.js

```diff
@@ -17,6 +17,20 @@ module.exports = function (_path) {
             filename: '[name].[chunkhash].js'
         },
         plugins: [
+        // Return error code when build fails.

```

Comment:
I don't feel strongly either way. My vote for using the plugin was to capture any upstream fixes (and ideally make it easier to gut this change when we move to Webpack 2.0, which apparently changes the default behavior).